content/docs/traffic-management/http2.md

```diff
@@ -0,0 +1,380 @@
+---
+title: HTTP/2
```

Comment:
I am wondering, would the HTTP/2 page be better placed under Destination Types?

<img width="1039" alt="image" src="https://github.com/user-attachments/assets/ae7b130e-0eba-41d9-b2bd-dda27edfc442" />